myuw/static/js/handlebars-helpers.js

```diff
@@ -425,3 +425,16 @@ Handlebars.registerHelper('slugify', function(value) {
     var slug = value.replace(/[^\w\s]+/gi, '').replace(/ +/gi, '-');
     return slug.toLowerCase();
 });
+
+Handlebars.registerHelper('phone_number', function(value) {
```

Comment:
Can you add some tests to directly test this?  I see there's an indirect test below, but if there are multiple input formats it would be good to make sure they're all being parsed.  And probably good for a fall-through test, just to be complete.